dune

```diff
@@ -18,6 +18,10 @@
 ;
 ; (_ (flags :standard -rectypes)))
 
+(alias
+ (name default)
+ (deps (alias install)))
```

Comment:
I would be more radical and make the default alias equate the `world` target, i.e. only `coq-core.install coq-stdlib.install coqide-server.install coqide.install` (non-doc install targets).